addons/point_of_sale/models/pos_order.py

```diff
@@ -602,8 +672,15 @@ def unlink(self):
 
     @api.model
     def create(self, values):
-        session = self.env['pos.session'].browse(values['session_id'])
-        values = self._complete_values_from_session(session, values)
+        if values.get('session_id'):
```

Comment:
Bad rebase, remove this diff, and use '_complete_values_from_session' that set the setdefault